mllib/src/test/java/org/apache/spark/mllib/regression/JavaStreamingLinearRegressionSuite.java

```diff
@@ -0,0 +1,80 @@
+/*
```

Comment:
Do you want to add a similar `JavaStreamingLogisticRegressionSuite`? Given that the primary change here is to the base class (used by both linear and logistic regression), might make sense to include in the same PR.